README.md

```diff
@@ -45,9 +45,9 @@ For example:
 
 ## Usage example
 
-[Simple Code Example](https://github.com/vesoft-inc/nebula-go/tree/master/basic_example/graph_client_basic_example.go)
+[Simple Code Example](https://github.com/vesoft-inc/nebula-go/blob/release-3.1/basic_example/graph_client_basic_example.go)
 
```

Comment:
Please update the link to direct to master branch. i.g. https://github.com/vesoft-inc/nebula-go/blob/master/examples/basic_example/graph_client_basic_example.go

Reply:
Thanks for the reminder, I've updated the two correct links directly to the master branch, please check it.